libs/fabric/package.json

```diff
@@ -9,38 +9,7 @@
         "react": "React",
         "react-dom": "ReactDOM",
         "@angular-react/core": "ɵa",
-        "office-ui-fabric-react/lib/Breadcrumb": "Breadcrumb",
-        "office-ui-fabric-react/lib/Button": "Button",
-        "office-ui-fabric-react/lib/Calendar": "Calendar",
-        "office-ui-fabric-react/lib/Callout": "Callout",
-        "office-ui-fabric-react/lib/Checkbox": "Checkbox",
-        "office-ui-fabric-react/lib/ChoiceGroup": "ChoiceGroup",
-        "office-ui-fabric-react/lib/ComboBox": "ComboBox",
-        "office-ui-fabric-react/lib/CommandBar": "CommandBar",
-        "office-ui-fabric-react/lib/DatePicker": "DatePicker",
-        "office-ui-fabric-react/lib/DetailsList": "DetailsList",
-        "office-ui-fabric-react/lib/Dialog": "Dialog",
-        "office-ui-fabric-react/lib/Divider": "Divider",
-        "office-ui-fabric-react/lib/Fabric": "Fabric",
-        "office-ui-fabric-react/lib/GroupedList": "GroupedList",
-        "office-ui-fabric-react/lib/HoverCard": "HoverCard",
-        "office-ui-fabric-react/lib/Icon": "Icon",
-        "office-ui-fabric-react/lib/Image": "Image",
-        "office-ui-fabric-react/lib/Link": "Link",
-        "office-ui-fabric-react/lib/MarqueeSelection": "MarqueeSelection",
-        "office-ui-fabric-react/lib/MessageBar": "MessageBar",
-        "office-ui-fabric-react/lib/Modal": "Modal",
-        "office-ui-fabric-react/lib/Panel": "Panel",
-        "office-ui-fabric-react/lib/Persona": "Persona",
-        "office-ui-fabric-react/lib/Pivot": "Pivot",
-        "office-ui-fabric-react/lib/SearchBox": "SearchBox",
-        "office-ui-fabric-react/lib/Shimmer": "Shimmer",
-        "office-ui-fabric-react/lib/Slider": "Slider",
-        "office-ui-fabric-react/lib/SpinButton": "SpinButton",
-        "office-ui-fabric-react/lib/Spinner": "Spinner",
-        "office-ui-fabric-react/lib/TextField": "TextField",
-        "office-ui-fabric-react/lib/Toggle": "Toggle",
-        "office-ui-fabric-react/lib/Tooltip": "Tooltip"
+        "office-ui-fabric-react": "Fabric"
```

Comment:
@shadowglen I'd appreciate some details as to how this improves tree-shaking.

Reply:
What I do here is not to improve tree-shaking itself.
Actually the original direct-path references were used because Fabric did not support tree-shaking before v6 - if any component was referred through package root, the entire package will be bundled by Webpack.
As we are moving to Webpack 4+ and Fabric v6, the direct-path reference is unnecessary now, so I unify them to package root.
Therefore, if Fabric is preloaded in the website, we can just make it as external module so that it won’t be bundled by Webpack.